config/settings.yml.example

```diff
@@ -3,7 +3,12 @@
 :database:
 
 # URL of the smart proxy, used for reporting back
-:callback_url: 'https://localhost:8443'
+:callback_url: 'http://localhost:9292'
```

Comment:
Why `9292`? Isn't proxy listening on `8443` for https and `8000` for http by default?